src/default_metrics.go

```diff
@@ -0,0 +1,127 @@
+	"net.maxUsedConnections":                      {"Max_used_connections", metric.GAUGE},
+	"net.threadsConnected":                        {"Threads_connected", metric.GAUGE},
+	"net.threadsRunning":                          {"Threads_running", metric.GAUGE},
+	"query.comCommitPerSecond":                    {"Com_commit", metric.RATE},
+	"query.comDeletePerSecond":                    {"Com_delete", metric.RATE},
+	"query.comDeleteMultiPerSecond":               {"Com_delete_multi", metric.RATE},
+	"query.comInsertPerSecond":                    {"Com_insert", metric.RATE},
+	"query.comInsertSelectPerSecond":              {"Com_insert_select", metric.RATE},
+	"query.comReplaceSelectPerSecond":             {"Com_replace_select", metric.RATE},
+	"query.comRollbackPerSecond":                  {"Com_rollback", metric.RATE},
+	"query.comSelectPerSecond":                    {"Com_select", metric.RATE},
+	"query.comUpdatePerSecond":                    {"Com_update", metric.RATE},
+	"query.comUpdateMultiPerSecond":               {"Com_update_multi", metric.RATE},
+	"db.handlerRollbackPerSecond":                 {"Handler_rollback", metric.RATE},
+	"query.preparedStmtCountPerSecond":            {"Prepared_stmt_count", metric.RATE},
+	"query.queriesPerSecond":                      {"Queries", metric.RATE},
+	"query.questionsPerSecond":                    {"Questions", metric.RATE},
+	"query.slowQueriesPerSecond":                  {"Slow_queries", metric.RATE},
+	"db.innodb.bufferPoolPagesData":               {"Innodb_buffer_pool_pages_data", metric.GAUGE},
+	"db.innodb.bufferPoolPagesFree":               {"Innodb_buffer_pool_pages_free", metric.GAUGE},
+	"db.innodb.bufferPoolPagesTotal":              {"Innodb_buffer_pool_pages_total", metric.GAUGE},
+	"db.innodb.dataReadBytesPerSecond":            {"Innodb_data_read", metric.RATE},
+	"db.innodb.dataWrittenBytesPerSecond":         {"Innodb_data_written", metric.RATE},
+	"db.innodb.logWaitsPerSecond":                 {"Innodb_log_waits", metric.RATE},
+	"db.innodb.rowLockCurrentWaits":               {"Innodb_row_lock_current_waits", metric.GAUGE},
+	"db.innodb.rowLockTimeAvg":                    {"Innodb_row_lock_time_avg", metric.GAUGE},
+	"db.innodb.rowLockWaitsPerSecond":             {"Innodb_row_lock_waits", metric.RATE},
+	"db.openFiles":                                {"Open_files", metric.GAUGE},
+	"db.openTables":                               {"Open_tables", metric.GAUGE},
+	"db.openedTablesPerSecond":                    {"Opened_tables", metric.RATE},
+	"db.tablesLocksWaitedPerSecond":               {"Table_locks_waited", metric.RATE},
+	"software.edition":                            {"version_comment", metric.ATTRIBUTE},
+	"software.version":                            {"version", metric.ATTRIBUTE},
+	"cluster.nodeType":                            {"node_type", metric.ATTRIBUTE},
+}
+
+var defaultMetrics560 = map[string][]interface{}{
```

Comment:
> Is this only for version 5.6.0? I'd assume this will be same for 5.7.x also. If so, can we update the name to be a bit more descriptive. Right now it seems that this will only work for 5.6.0.

Same here.

Reply:
Resolved it in this [commit](https://github.com/newrelic/nri-mysql/pull/170/commits/fe29f92e5ad2d923fd031d278d2ecb482b0e69be)